tests/extras/test_translator.py

```diff
@@ -17,23 +17,31 @@
 }
 
 
+@pytest.mark.parametrize("translator", ("rest_tlr","tlr"))
```

Comment:
Well, that's new to me! Nice!

Minor comment: `translator` is really a translator selector or name, not a Translator instance. Please consider a different name to make that clearer. `tlr_fixture_name`?